stellarphot/core.py

```diff
@@ -166,9 +166,15 @@ def _update_colnames(self, colname_map, data):
     def _update_passbands(self):
         # Converts filter names in filter column to AAVSO standard names
         # Assumes _passband_map is in namespace.
-        for orig_pb, aavso_pb in self._passband_map.items():
-            mask = self["passband"] == orig_pb
-            self["passband"][mask] = aavso_pb
+
+        # Create a list of new passband names instead of trying to change names in place
+        # in case any of the new names are longer than the longest of the old names.
+        # If that happens, astropy by default just truncates the names.
```

Comment:
I assume you mean that the truncation is a bad thing...

Also, took me a bit of effort to comprehend the list comprehension... but I think it is sound assuming what you want is for all the `orig_pb` to be kept, using the existing `_passband_map` assigned to `orig_pb` if there is one, otherwise using `orig_pb` itself (I assume instead of letting astropy truncate the old name).